make/goreleaser.mk

```diff
@@ -8,6 +8,11 @@ ifndef GORELEASER_CURRENT_TAG
 export GORELEASER_CURRENT_TAG=$(GIT_TAG)
 endif
 
+.PHONY: docker-buildx
+docker-buildx: ## Creates buildx builder container that supports multiple platforms.
+docker-buildx:
+	 docker buildx create --use --name=caren --platform=linux/arm64,linux/amd64 || true
```

Comment:
Won't the use flag here cause all my other (non CAREN) image builds to also use this builder? If we go this way, then perhaps it's better not to add the use here and instead specify the builder to use in goreleaser config (https://goreleaser.com/customization/docker/#use-a-specific-builder-with-docker-buildx)?

Reply:
that's a good idea, i'll add that to the config.